lib/vagrant-ovirt4/action/create_network_interfaces.rb

```diff
@@ -27,8 +27,18 @@ def call(env)
             iface_options = scoped_hash_override(options, :ovirt)
           end
 
+          config = env[:machine].provider_config
+          @logger.info("Finding network #{iface_options[:network_name]} for given cluster #{config.cluster}")
+          network_profile_id = nil
+          clusters_service = env[:connection].system_service.clusters_service
+          cluster = clusters_service.list(search: "name=#{config.cluster}").first
```

Comment:
It depends on data centers, if two clusters are in the same DC, it is OK, but when you have two DCs with the same network name, it fails.